v2/sourcedb-to-spanner/src/main/java/com/google/cloud/teleport/v2/templates/PipelineController.java

```diff
@@ -256,6 +256,8 @@ static PipelineResult executeShardedMigration(
                       shard.getUserName(),
                       shard.getPassword(),
                       entry.getKey(),
+                      // TODO(thiagotnunes): Specify the namespace from a shard
+                      null,
```

Comment:
@VardhanThigle I am not specifying this at the moment, but I could. Should I address this in a future PR or now?

Reply:
You can take it up in future PR.